pages/chat.tsx

```diff
@@ -27,8 +29,11 @@ export default function Chat() {
   const [faithfulnessScore, setFaithfulnessScore] = useState(0);
   const [contextRelevanceScore, setContextRelevanceScore] = useState(0);
   const [answerRelevanceScore, setAnswerRelevanceScore] = useState(0);
-  const [uploadedFileNames, setUploadedFileNames] = useState([]);
-  const [selectedFiles, setSelectedFiles] = useState([]);
+  const [uploadedFiles, setUploadedFiles] = useState([]);
+  const [context, setContext] = useState<string[] | null>(null);
```

Comment:
can u use a a more descriptive naming, unclear what exactly "setContext" refers to